bindings/java/mongocrypt/build.gradle.kts

```diff
@@ -211,14 +227,7 @@ publishing {
 
             artifact(tasks["sourcesJar"])
             artifact(tasks["javadocJar"])
-            versionMapping {
```

Comment:
This removes the resolved version mapping that was the cause of the pom.xml containing the resolved issue. Because it now uses the default declared version resolution, there is no need to modify the pom.xml.

https://docs.gradle.org/current/userguide/publishing_maven.html#publishing_maven:resolved_dependencies